tests/toranj/test-013-off-mesh-route-traffic.py

```diff
@@ -132,7 +134,23 @@
 fed1.add_route(OFF_MESH_ROUTE_3)
 fed1.add_ip6_address_on_interface(OFF_MESH_ADDR_3)
 
-time.sleep(0.5)
+# Wait till network data is updated on all nodes and they all see all
```

Comment:
Comment says "all nodes", yet `fed1` is not included in the check below. Should it be?

Reply:
We can add it but I excluded  it mainly since we only care about the nodes that will be initiating tx and sending traffic to one of the off-mesh routes. Let me update the comment to be clear about this.